pages/src/navigation/sidebar/sidebar.less

```diff
@@ -23,6 +23,10 @@ esl-d-sidebar {
   }
 
   @media @xs-sm {
+    position: fixed;
+    top: 56px;
+    height: calc(100vh - 56px);
+
```

Comment:
Mobile settings is the default settings applied to an element, it produces code duplication in case we overwrite it in the media query